io.github.snesrev.Zelda3.yml

```diff
@@ -0,0 +1,66 @@
+app-id: io.github.snesrev.Zelda3
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: zelda3.py
+
+finish-args:
+ - --device=all
+ - --share=ipc
+ - --socket=x11
+ - --socket=pulseaudio
+modules:
+ - name: python3-pillow
+   buildsystem: simple
+   build-commands:
+    - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "pillow" --no-build-isolation
+   sources:
+    - type: file
+      url: https://files.pythonhosted.org/packages/8c/92/2975b464d9926dc667020ed1abfa6276e68c3571dcb77e43347e15ee9eed/Pillow-9.2.0.tar.gz
+      sha256: 75e636fd3e0fb872693f23ccb8a5ff2cd578801251f3a4f6854c6a5d437d3c04
+   
+ - name: python3-pyyaml
+   buildsystem: simple
+   build-commands:
+    - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "pyyaml" --no-build-isolation
+   sources:
+    - type: file
+      url: https://files.pythonhosted.org/packages/36/2b/61d51a2c4f25ef062ae3f74576b01638bebad5e045f747ff12643df63844/PyYAML-6.0.tar.gz
+      sha256: 68fb519c14306fec9720a2a5b45bc9f0c8d1b9c72adf45c37baedfcd949c35a2
+ 
+ - name: zelda3
+   buildsystem: simple
+   build-commands:
+    - sed -i '/(TARGET_EXEC) tables\/zelda3_assets.dat/d' Makefile
+    - make
+   post-install:
+    - install -Dm755 zelda3 ${FLATPAK_DEST}/bin/zelda3
+    - install -Dm755 zelda3.py ${FLATPAK_DEST}/bin/zelda3.py
+    - cp -R . ${FLATPAK_DEST}/bin/src
+    - install -Dm644 io.github.snesrev.Zelda3.appdata.xml -t ${FLATPAK_DEST}/share/metainfo/
+    - install -Dm644 io.github.snesrev.Zelda3.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/io.github.snesrev.Zelda3.svg
+    - |
+        for s in 16 22 24 32 48 64 128 256; do
```

Comment:
you do not need this at all.

Reply:
On my system, the svg wasn't rendering quite right. that's why I added the pngs.

Reply:
done